Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -144,53 +144,54 @@ static void APIENTRY ClearDepthf(GLfloat depthval)
 
 static void InitDriverInfo()
 {
-  std::string svendor = std::string(g_ogl_config.gl_vendor);
-  std::string srenderer = std::string(g_ogl_config.gl_renderer);
```

Comment:
These can be changed to std::string_view instances, eliminating the need for C-style string functions.